js/components/interface/graph-visualization/README.md

```diff
@@ -0,0 +1,93 @@
+# Geppetto Graph Visualizer (GGV)
+
+## Parameters
+
+- **data**:
+
+```javascript
+{
+    nodes: [
+        {id: 1},
+        {id: 2},
+        {id: 3}
+    ],
+    links: [
+        {source: 1, target: 2},
+        {source: 2, target: 3},
+        {source: 3, target: 1}
+    ]
+}
+```
+
+## With react-flexlayout
+
+If you are planning to use GGV with react-flexlayout, remember to add the resize event listener to the node where GGV will be attached. Example bellow:
```

Comment:
We probably don't need this with the suggested change (move logic to render)

Reply:
Removed it